searcch_backend/migrations/versions/f29a393d7f66_add_user_ipd_credential_table.py

```diff
@@ -0,0 +1,32 @@
+"""add user_ipd_credential table
+
+Revision ID: f29a393d7f66
+Revises: b11c5baa16f9
+Create Date: 2023-02-20 12:50:55.824852
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = 'f29a393d7f66'
+down_revision = 'b11c5baa16f9'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table('user_idp_credentials',
```

Comment:
On the call I suggested that we do something more general, but on further reflection this is good for right now.  Sorry.